CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set(PROJECT_NAME Covid19EERAModel)
 
 project(${PROJECT_NAME} VERSION 0.10.0 LANGUAGES CXX)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
What's the reason for requiring C++17 support?

My concern would be that only relatively up-to-date compilers support it: people working on older systems may struggle to build this code if we require 17. But if we need a feature from it, then it's okay.

Reply:
The datapipeline.hh file is currently using std::is_same_v which is a C++17 feature.